libtransact/src/state/merkle/sql/operations/update_index.rs

```diff
@@ -148,10 +148,10 @@ impl<'a> MerkleRadixUpdateIndexOperation for MerkleRadixOperations<'a, SqliteCon
 
 #[cfg(feature = "sqlite")]
```

Comment:
Should this feature guard be removed in this commit?

Reply:
Looks like it's currently removed here: https://github.com/hyperledger/transact/pull/166/commits/17769ad91673c301afe1beeb72426f675dd62ee6

Reply:
Moved is to the appropriate commit